wgpu-hal/src/vulkan/device.rs

```diff
@@ -1120,14 +1120,12 @@ impl crate::Device<super::Api> for super::Device {
                 .compare_op(conv::map_comparison(fun));
         }
 
-        if self
-            .shared
-            .downlevel_flags
-            .contains(wgt::DownlevelFlags::ANISOTROPIC_FILTERING)
-        {
+        if desc.anisotropy_clamp != 1 {
+            // We only enable the downlevel flag if supports 16x anisotropy,
```

Comment:
```suggestion
            // We only enable anisotropy if it is supported,
```

Not sure if the wording is better (just a suggestion).